server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java

```diff
@@ -18,16 +18,6 @@
  */
 package org.apache.iotdb.db.query.reader.series;
 
```

Comment:
avoid more changes like this in the future.

Reply:
Ok